pkgs/by-name/ko/korrect/package.nix

```
@@ -0,0 +1,24 @@
+{
+  lib,
+  fetchCrate,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage (finalAttrs: {
+  pname = "korrect";
+  version = "0.1.1";
+
+  src = fetchCrate {
+    inherit (finalAttrs) pname version;
+    hash = "sha256-LS3pzDIb1G6lrP8lAG/Yd7Zcehjm5/TNEvTI+MnOomk=";
+  };
+  cargoHash = "sha256-vpjldD+aVVgCMQ/n+WZDPMMBRFPEeQZa09b45Q3m5UM=";
+
+  # TODO updatescript
+  meta = {
+    description = "A kubectl version managing shim that invokes the correct kubectl version ☸ ";
```

Please avoid articles at the beginning of description and symbols :)

```suggestion
    description = "Kubectl version managing shim that invokes the correct kubectl version";
```